certbot/certbot/_internal/plugins/manual.py

```diff
@@ -162,8 +173,12 @@ def _perform_achall_manually(self, achall, lastchall=False):
             if self.subsequent_dns_challenge:
                 # 2nd or later dns-01 challenge
                 msg += self._SUBSEQUENT_DNS_CHALLENGE_INSTRUCTIONS
+            elif self.subsequent_any_challenge:
```

Comment:
Looks like the reason this isn't necessary is that we sort the requests to have the dns challenges first. That being said, I could see us changing that in the future and forgetting to add this here, so let's leave it.

Reply:
I actually noticed that `certbot_test` ran locally against a local Pebbel (through `run_acme_server`) randomly listed the challenges in all possible ways, including HTTP-01 first.